pyproject.toml

```
@@ -16,14 +16,17 @@ python = "^3.8"
 requests = "^2"
 requests_oauthlib = "^1"
 msal = "^1"
+sortedcontainers = "^2.3"
+numpy = { version = "^1.18.5", optional = true }
```

Any reason we can't allow 1.0.0 to 1.18.5?
```suggestion
numpy = { version = "^1", optional = true }
```

---

Yup, gotta trust pandas devs 😅 (they have min version at `>=1.20.0`)

---

So I'm actually gonna bump it further to match